resources/user-guides/README.md

```diff
@@ -1,17 +1,48 @@
 # Tutorials
 
-## Implementing Experience Cloud solutions in Mobile Android and iOS Swift applications
+## Implement Adobe Experience Platform Mobile SDKs in mobile applications
 
-This sections provides a starting point for mobile application developers who want to learn how to implement Adobe Experience Cloud solutions in their applications. You can complete the following sample Swift or Android tutorials to learn about implementing Experience Cloud in a controlled environment:
+This tutorial guides you through the implementation of the Platform Mobile SDK in a sample retail app called Luma. The Luma app has functionality that lets you build a realistic implementation. The lessons are designed for iOS and written in Swift, but many of the concepts also apply to Android.
 
-* [Implementing the Experience Cloud in Mobile iOS Swift Applications](https://experienceleague.adobe.com/docs/launch-learn/implementing-in-mobile-ios-swift-apps-with-launch/index.html)
-* [Implementing the Experience Cloud in Mobile Android Applications](https://experienceleague.adobe.com/docs/launch-learn/implementing-in-mobile-android-apps-with-launch/index.html)
+* [Implement Platform Mobile SDKs in mobile applications](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/overview.html)
+
+## Getting started with Adobe Experience Platform
+
+This tutorial illustrates how you may send Experience Data Model (XDM) commerce events to Adobe Experience Platform via Experience Edge using the AEP Edge extension in a sample application, provided to you in iOS (Swift) and Android.
+
+* [Getting started with Adobe Experience Platform](https://aep-sdks.gitbook.io/docs/resources/user-guides/getting-started-with-adobe-experience-platform)
 
 ## Tracking Beacons in Mobile Android and iOS applications
 
 Sending beacon data to Adobe Analytics for reporting can be an effective to understand how your audience can interact with physical landmarks. You can manually send beacon tracking data to your Analytics server and create rules based on a user's proximity to your beacons. This solution relies on the Profile extension to generate beacon-related rules.
 
-For more information, see the following link:
-
 * [Implementing beacon tracking for Adobe Analytics in Mobile Android and iOS applications](track-beacon.md)
 
+## Attach or modify data in SDK events
+
+In this tutorial you will learn how to enable advanced use-cases with Attach Data and Modify Data actions.
```

Comment:
For proper parallelization, this should also start with "These tutorials illustrate how to..."